src/System.CommandLine/Help/HelpOption.cs

```diff
@@ -1,24 +1,19 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System.CommandLine.Invocation;
-using System.CommandLine.IO;
-using System.Threading;
-using System.Threading.Tasks;
-
 namespace System.CommandLine.Help
 {
-    internal class HelpOption : Option<bool>
+    public sealed class HelpOption : Option<bool>
     {
-        internal HelpOption(string name, string[] aliases)
+        public HelpOption(string name, string[] aliases)
             : base(name, aliases, new Argument<bool>(name) { Arity = ArgumentArity.Zero })
         {
             AppliesToSelfAndChildren = true;
             Description = LocalizationResources.HelpOptionDescription();
-            Action = new HelpOptionAction();
+            Action = new HelpAction();
         }
 
-        internal HelpOption() : this("--help", new[] { "-h", "/h", "-?", "/?" })
+        public HelpOption() : this("--help", new[] { "-h", "/h", "-?", "/?" })
```

Comment:
Do you want to introduce something like `HelpOption.DefaultAliases` and use it here for discoverability, etc.?

Reply:
How would it be used by the end users? Currently to get the default aliases they can just do `new HelpOption().Aliases`

Reply:
IIRC Krzysztof or Jeremy started this discussion and the context was that it was not clear what aliases the `HelpOption` will use when created and added to the `Command`.

In #2107 I've made the config mutable and simply added xml docs to the `HelpOption` ctor:

https://github.com/dotnet/command-line-api/blob/2dd5b76efa1770ee230107e3d732fa9e03796e09/src/System.CommandLine/Help/HelpOption.cs#L8-L18

I am expecting that this is going to solve the problem